lib/mwoffliner.lib.js

```diff
@@ -25,10 +25,11 @@ var os = require( 'os' );
 var crypto = require( 'crypto' );
 var unicodeCutter = require( 'utf8-binary-cutter' );
 var htmlMinifier = require('html-minifier');
+const fetch = require('node-fetch')
 
 module.exports = {
     getParametersList: () => [
-        { name: 'mwUrl', description: 'Mediawiki base URL', required: true },
+        { name: 'mwUrl', description: 'Mediawiki base URL. Dont forget the trailing /', required: true },
```

Comment:
Important indeed, but better make directly a check at the line : mwUrl =.... This is easier for user.